### PR TITLE
Update dependencies, address breaking USWDS change

### PR DESCRIPTION
**Pull request by hursey013**
Update dependencies to pull in latest USWDS, fixing the specified [breaking change](https://github.com/uswds/uswds/releases/tag/v2.13.0).

When I did a fresh install of this repo in order to rebase https://github.com/18F/federalist-uswds-gatsby/pull/467, I noticed USWDS was not building correctly. Updating USWDS led to several other required updates, including replacing `node-sass` with `sass` (which is the culprit of numerous snyk/dependabot PRs), so I went ahead and updated everything and fixed the different breaking changes specified by the Gatsby/USWDS docs.

### CODE REVIEW

**Comment by hursey013** [comment]
Working through an issue that seems to stem from `remark-mdx`...

**Comment by hursey013** [comment]
Working through an issue that seems to stem from `remark-mdx`...

**Comment by hursey013** [comment]
@18F/federalist-admins ready for another look 👀

File: src/components/search-form.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import { navigate } from '@reach/router';
+import search from '../../node_modules/uswds/dist/img/usa-icons-bg/search--white.svg';
```
Comment (apburnes) [inline src/components/search-form.js]: You could pull in the svg from the package directly.

```suggestion
import search from 'uswds/src/img/usa-icons-bg/search--white.svg';
```

Comment (hursey013) [reply src/components/search-form.js]: @apburnes Not sure why but the other components were using that relative path syntax, but I went ahead and updated them as you suggested for consistency: https://github.com/18F/federalist-uswds-gatsby/pull/474/commits/bf9a86cfb6ee7f94afd346d45b28616604b7701f

Comment (apburnes) [reply src/components/search-form.js]: 💯

**Review by apburnes — APPROVED** [review]
👍

**Pull request merged**